desktopexporter/internal/server/server.go

```diff
@@ -23,29 +22,25 @@ var assets embed.FS
 type Server struct {
 	server http.Server
 	Store  *store.Store
+	isDev  bool
 }
 
-func NewServer(endpoint string, dbPath string) *Server {
+func NewServer(endpoint string, dbPath string, isDev bool) *Server {
```

Comment:
I don't think we're passing `isDev` to the created `Server` object? In that case it will default to `false`

Reply:
I was not. I am now. 😅